src/main/java/uk/gov/laa/ccms/caab/controller/application/summary/EditGeneralDetailsSectionController.java

```diff
@@ -364,26 +394,209 @@ private void populateLinkedCasesConfirmDropdowns(final Model model) {
         .build();
   }
 
+  /**
+   * Handles the GET request for searching linked cases.
+   *
+   * @param userDetails The details of the logged-in user.
+   * @param model       The UI model to add attributes to.
+   * @param session     The HTTP session object.
+   * @return String     The view name for linked cases search.
+   */
+  @GetMapping("/application/summary/linked-cases/search")
+  public String linkedCasesSearchGet(
+      @SessionAttribute(USER_DETAILS) final UserDetail userDetails,
+      final Model model,
+      final HttpSession session) {
+
+    populateLinkedCasesSearchDropdowns(userDetails, model);
+
+    final CaseSearchCriteria caseSearchCriteria = new CaseSearchCriteria();
```

Comment:
This will wipe the search criteria if the user clicks the back button on the search results screen.
Did we not want to do it like the others and have CaseSearchCriteria in the class-level SessionAttributes?

Reply:
sorted, the criteria will reset on the linked cases summary page, returning to the page after a search will keep the data